Extension/package.json

```diff
@@ -2685,7 +2685,7 @@
       "integrity": "186614524A13EC75BF68798861818D25CE82FDE293C57B725C11F62B0694EC55"
     },
     {
-      "description": "C/C++ language components (OS X)",
+      "description": "C/C++ language components (macOS x86_64)",
```

Comment:
Should the x86_64 be x64 instead for macOS?

Reply:
I'm not certain, but I see on [this developer documentation page](https://developer.apple.com/documentation/apple-silicon/building-a-universal-macos-binary) that apple refers to `x86_64`.